source/common/json/config_schemas.cc

```diff
@@ -954,6 +954,26 @@ const std::string Json::Schema::TOP_LEVEL_CONFIG_SCHEMA(R"EOF(
         "required" : ["type", "config"],
         "additionalProperties" : false
       },
+      "zipkin_driver" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["zipkin"]
```

Comment:
I might have missed some prior discussion, but my reading of this enum is that tracing drivers were originally envisioned to all be specified under a single "driver" schema, with a type ("zipkin", "lightstep") specified via this field.

Most of the other fields are identical across "lightstep_driver" and "zipkin_driver."